docker/test/base/setup_export_logs.sh

```diff
@@ -15,7 +15,7 @@ CLICKHOUSE_CI_LOGS_USER=${CLICKHOUSE_CI_LOGS_USER:-ci}
 # Pre-configured destination cluster, where to export the data
 CLICKHOUSE_CI_LOGS_CLUSTER=${CLICKHOUSE_CI_LOGS_CLUSTER:-system_logs_export}
 
-EXTRA_COLUMNS=${EXTRA_COLUMNS:-"pull_request_number UInt32, commit_sha String, check_start_time DateTime, check_name LowCardinality(String), instance_type LowCardinality(String), instance_id String, "}
+EXTRA_COLUMNS=${EXTRA_COLUMNS:-"pull_request_number UInt16, commit_sha String, check_start_time DateTime('UTC'), check_name String, instance_type String, instance_id String, "}
```

Comment:
```
In [16]: 1 << 16
Out[16]: 65536
```

PR numbed: #55503

pr number #50000: May 22

I feel it's *very* bad idea

Reply:
I will fix it ...

Reply:
What about calculating the hash out of `EXTRA_COLUMNS` automatically too? The manual increment feels poor there